src/assets/styles/_spacing.scss

```diff
@@ -0,0 +1,29 @@
+%component-spacing {
+  /* Component spacing variables */
+
+  --calcite-icon-size: 1rem;
+
+  /**
+  * These will be deprecated. (o,,o)
+  */
+  --calcite-spacing-eighth: theme("spacing.05");
```

Comment:
Interesting, it looks like the spacing scale has recently been [expanded](https://github.com/tailwindlabs/tailwindcss/pull/2609/commits/559bf9589821ccf0773d4d727a6523028736ad91). Looking at the new values, and judging from the commit message, it might be worth discussing constraining the options to those of the former scale.

Reply:
Thanks for looking at this.
This vars placeholder will go away once all the components using it are refactored to use Tailwind.